.taskcluster.yml

```diff
@@ -62,6 +62,10 @@ tasks:
         command: yarn lint
       - name: test:meta
         command: yarn test:meta
+      - $if: 'tasks_for == "github-push"'  # only do cleanup on pushes; no sense doing so on pull requests
```

Comment:
The only thing to note here is that when we push to this repo and make a PR, the actual tasks that are triggered are for the push. The only time PR action is triggered is if there's a PR from another repo. Maybe make this so that it is only on pushes to master or something?

Reply:
Oh my, that's pretty clever -- I didn't see that.  Yeah, I'll update to only trigger on pushes to master.